orchestra/frontend/src/assets/ShuffleIcon.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const ShuffleIcon = () => {
+  return (
+    <svg width="24px" height="18px" viewBox="0 0 24 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlnsXlink="http://www.w3.org/1999/xlink">
+      <g id="Dashboard-–-Tabular" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
+          <g transform="translate(-30.000000, -25.000000)" fill="#695FF6" fill-rule="nonzero" id="Header">
+              <g>
+                  <g id="Group">
+                      <g transform="translate(30.000000, 23.000000)">
+                          <g id="shuffle-98" transform="translate(0.000000, 2.000000)">
+                              <path d="M0,5 L4.219,5 C5.069,5 5.881,5.362 6.448,5.993 L7.809,7.505 L9.154,6.01 L7.934,4.654 C6.989,3.604 5.635,3 4.219,3 L0,3 L0,5 Z" id="Path"></path>
+                              <path d="M24,4 L19,0 L19,3 L16.781,3 C15.365,3 14.01,3.604 13.065,4.655 L6.448,12.007 C5.881,12.638 5.068,13 4.219,13 L0,13 L0,15 L4.219,15 C5.635,15 6.99,14.396 7.935,13.345 L14.552,5.993 C15.119,5.362 15.932,5 16.781,5 L19,5 L19,8 L24,4 Z" id="Path"></path>
+                              <path d="M24,14 L19,10 L19,13 L16.781,13 C15.931,13 15.119,12.638 14.552,12.007 L13.191,10.495 L11.846,11.99 L13.066,13.345 C14.011,14.397 15.366,15 16.782,15 L19,15 L19,18 L24,14 Z" id="Path"></path>
+                          </g>
+                      </g>
+                  </g>
+              </g>
+          </g>
+      </g>
+    </svg>
+  )
+}
+
+export default ShuffleIcon
```

Comment:
newline

Reply:
ok i'll stop mentioning newline -> there are a bunch of files in this PR that need them :)